dist/common/scripts/scylla_io_setup

```diff
@@ -122,6 +122,14 @@ class scylla_cpuinfo:
             return len(self._cpu_data["system"])
 
 def run_iotune():
+            with open("/etc/sysctl.conf", "r+") as file:
+                for line in file:
+                    if 'aio-max-nr' in line:
+                        break
+                    else: # not found, we are at the eof
+                        file.write("fs.aio-max-nr = 30000000\n") # append missing data
+                        os.system('sysctl -p /etc/sysctl.conf')
```

Comment:
This is generic code so let's be extra careful. Why is this change needed? Perhaps make it a separate pull request.

Reply:
I will remove it as i found there is a function `configure_aio_slots` under `scylla_prepare` .